docs/source/usage/tutorials/using_vpg.md

```diff
@@ -0,0 +1,73 @@
+# Vanilla Policy Gradient (VPG)
+
+If you wanted to explore Policy Gradient algorithms in RL, there is a high chance you would've heard of PPO, DDPG, etc. but understanding them can be tricky if you're just starting.
+
+VPG is arguably one of the easiest to understand policy gradient algorithms while still performing to a good enough level
+
+Let's understand policy gradient at a high level, unlike the classical algorithms like Q-Learning, Monte Carlo where you try to optimise the outputs of the action-value function of the agent which are then used to determine the optimal policy. In policy gradient, as one would like to say we go directly for the kill shot, basically we optimise the thing we want to use at the end, i.e. the Policy.
+
+So that explains the "Policy" part of Policy Gradient, so what about "Gradient", so gradient comes from the fact that we try to optimise the policy by gradient ascent (unlike the popular gradient descent, here we want to increase the values, hence ascent). So that explains the name, but how does it even work.
+
+For that, have a look at the following Psuedo Code (source: [OpenAI](spinningup.openai.com))
```

Comment:
You can add https://spinningup.openai.com/en/latest/spinningup/rl_intro3.html as a source for more detailed derivation also.

Reply:
Sure